appshell/node-core/Server.js

```diff
@@ -199,7 +199,20 @@ maxerr: 50, node: true */
                         stop();
                     });
                     
-                    wsServer = new WebSocket.Server({server: httpServer});
+                    wsServer = new WebSocket.Server({
+                        server: httpServer,
+                        verifyClient : function (info, callback) {
+                            // Accept connections originated from local system only
+                            // Also do a loose check on user-agent to accept connection only from Brackets CEF shell
+                            if (info.origin === "file://" && info.req.headers["user-agent"].indexOf(" Brackets") !== -1) {
```

Comment:
@petetnt Even I don't quire like the check , you can suggest something which looks better :smile: